### ISSUE TITLE
Editing resorce type that's represented by include cause editing main file

### ISSUE THREAD

**Issue by ddenisenko** [mulesoft/api-workbench#6]
This bug works equally for traits and for resource types.
1) Open GitHub project: https://github.com/raml-apis/GitHub/tree/raml1.0
2) Select resource type reference item in the `resourceTypes:` partition.
3) Go to tab "Pallete" of Details view.
4) Create method.
5) Also user can specify fields at the tab "References"
Result: 
Resource type reference item has a method and empt field from tab "References".

```
  - base: !include resourceTypes/base.raml
      get:
      securedBy: 
      type: 
      is: 
```
IDE throws exception:

[Enter steps to reproduce below:]

**Atom Version**: 1.0.19
**System**: Mac OS X 10.11
**Thrown From**: 'api-workbench' package, v0.7.0

### Stack Trace

Uncaught Insert into mapping with INCLUDE_REF value not supported

```
At /Users/vasil/Documents/api-workbench/api-workbench/src/raml1/jsyaml/jsyaml2lowLevel.js:1391

undefined
```

### Commands

```
     -5:47.7.0 core:backspace (atom-text-editor.editor.is-focused)
  2x -4:54.7.0 core:undo (select.form-control)
```

### Config

```json
{
  "core": {}
}
```

### Installed Packages

```coffee
# User
api-workbench, v0.7.0
linter, v1.8.1

# Dev
No dev packages
```

**Comment by ddenisenko** [comment]
Relevant comments:

alexanderosipov :
wrong behavior fixed, now all changes made in included unit
left to fix: modified include doesn't reflect in editor (resourceTypes/base.raml in this certain case)

**Comment by ddenisenko** [comment]
Fixed. Fix should be available in 0.8.7.
